android/app/src/main/java/poke/rogue/helper/presentation/type/typeselection/TypeSelectionBottomSheetFragment.kt

```diff
@@ -0,0 +1,53 @@
+package poke.rogue.helper.presentation.type.typeselection
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import poke.rogue.helper.data.mapper.toSelectionUiModel
+import poke.rogue.helper.databinding.FragmentTypeChoiceBottomSheetBinding
+import poke.rogue.helper.local.dao.TypeDao
+import poke.rogue.helper.presentation.util.view.GridSpacingItemDecoration
+import poke.rogue.helper.presentation.util.view.dp
+
+class TypeSelectionBottomSheetFragment : BottomSheetDialogFragment() {
+    private var _binding: FragmentTypeChoiceBottomSheetBinding? = null
+    private val binding get() = _binding!!
```

Comment:
`!!` 보다는 예외를 정의해 넘겨주는건 어떨까요?
```suggestion
    private val binding get() = _binding ?: error("error msg..")
```

Reply:
넵 `!!` 대신 `requireNonNull` 이런식으로 수정해보겠습니다~